qual_id/pattern.py

```diff
@@ -39,6 +40,7 @@ class Pattern:
       "searchengine": SearchEngine(),
       "adjective": Adjective(),
       "city": City(),
+      "planet": Planet()
```

Comment:
Here's the offending line
```suggestion
      "planet": Planet(),
```